app/models/ticket_request.rb

```diff
@@ -253,8 +253,13 @@ def refunded?
     status == STATUS_REFUNDED
   end
 
+  def mark_refunded
+    Rails.logger.info { "ticket request marking refunded: #{id}" }
+    update status: STATUS_REFUNDED
```

Comment:
https://apidock.com/rails/ActiveRecord/Persistence/update

"Updates the attributes of the model from the passed-in hash and saves the record, all wrapped in a transaction. If the object is invalid, the saving will fail and false will be returned."

So this method would return false if the update false, or true if it succeeds.